 -W option breaks studio compiler
Internal test-build only. Will not be proposed to community

--- a/tests/libtest/Makefile.am	2017-10-10 07:07:02.636502997 +0000
+++ b/tests/libtest/Makefile.am	2017-10-10 07:07:26.868787058 +0000
@@ -115,7 +115,7 @@
 
 libstubgss_la_CPPFLAGS = $(AM_CPPFLAGS)
 libstubgss_la_LDFLAGS = $(AM_LDFLAGS) -avoid-version -rpath /nowhere
-libstubgss_la_CFLAGS = $(AM_CFLAGS) -g -Wno-unused-parameter
+libstubgss_la_CFLAGS = $(AM_CFLAGS) -g
 
 libstubgss_la_SOURCES = stub_gssapi.c stub_gssapi.h
 
